core/domain/stats_services.py

```diff
@@ -1117,6 +1117,54 @@ def get_learner_answer_details(entity_type, state_reference):
     return None
 
 
+def get_learner_answer_info_for_exploration(exp_id):
+    """Returns the learner answer info in dict format.
+
+    Args:
+        exp_id: str. The ID of the exploration.
+
+    Returns:
+        learner_answer_info_dict_list: list. Consist of dicts
```

Comment:
Please describe the structure of the learner answer info dicts, similar to the docstring here: https://github.com/oppia/oppia/blob/develop/core/jobs.py#L1606-L1629.

Please copy/paste the layout in all other places which return the same dicts.

Reply:
Done